core/src/main/scala/org/apache/spark/sql/SnappyDDLParser.scala

```diff
@@ -699,6 +708,41 @@ private[sql] case class TruncateTable(ifExists: Any,
   }
 }
 
+private[sql] case class AlterTableAddColumn(tableIdent: TableIdentifier,
+   addColumn: StructField) extends ExecuteCommand {
+
+  override def run(session: SparkSession): Seq[Row] = {
+    val snc = session.asInstanceOf[SnappySession]
+    val catalog = snc.sessionState.catalog
+    snc.alterTable(catalog.newQualifiedTableName(tableIdent), true, addColumn)
+    Seq.empty
+  }
+}
+
+private[sql] case class AlterTableDropColumn(
+  tableIdent: TableIdentifier, column: String) extends ExecuteCommand {
+
+  override def run(session: SparkSession): Seq[Row] = {
+    val snc = session.asInstanceOf[SnappySession]
+    val catalog = snc.sessionState.catalog
+    val plan = try {
+      snc.sessionCatalog.lookupRelation(tableIdent)
```

Comment:
drop column just specifies the name of the column and not it's type, hence the lookup in this case. Also the alterTablePIs are consistent they expect a complete StructField is passed in both add and drop cases.